lib/hijack/wrap_session.js

```diff
@@ -101,6 +102,8 @@ export function wrapSession (sessionProto) {
       response = Kadira.env.kadiraInfo.withValue(kadiraInfo, function () {
         return orginalSubHandler.call(self, msg, unblock);
       });
+      Kadira.models.pubsub.trackWaitedOn(kadiraInfo.trace.name, this.inQueue);
```

Comment:
You had removed this code for pubs. Does this also need to be removed for methods?

Reply:
good point, I removed it for the methods, but unblock is called for both (sub/method) at the end regardless of the output.

This line here: https://github.com/meteor/meteor/blob/release-3.0-2.14/packages/ddp-server/livedata_server.js#L656

So I will remove it for both